README.md

```diff
@@ -0,0 +1,51 @@
+# Login State
+
+Share Login State between the Sub Domains for Meteor Apps
+
+## How-To:
```

Comment:
Change this to `Getting Started`.

And also add a description on what user needs to in brief. May be just 2-3 lines.